rewrites.js

```diff
@@ -268,11 +264,15 @@ exports.REWRITES = [
   },
   // New recurring contributions page
   {
-    source: '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:slug/recurring-contributions',
+    source: '/:slug/recurring-contributions',
+    destination: '/recurring-contributions',
+  },
+  {
+    source: '/recurring-contributions',
     destination: '/recurring-contributions',
   },
   {
-    source: '/:parentCollectiveSlug?/:collectiveType(events|projects)?/:slug/subscriptions',
+    source: '/:slug/subscriptions',
```

Comment:
Is this ok in regard to your upcoming changes to routes with parents?

Reply:
Yes this is okay, in fact I did this change in my PR as well, https://github.com/opencollective/opencollective-frontend/pull/7395/files#. I think this should be changed and I added this `parentCollectiveSlug` part by mistake in an earlier commit (https://github.com/opencollective/opencollective-frontend/pull/7304/files). 🤔